posts/MarkoB/2017-02-06-validator-service-loader.adoc

```diff
@@ -4,21 +4,20 @@ Marko Bekhta
 :awestruct-layout: blog-post
```

Comment:
You forgot to change the package name in the new image you uploaded!

Maybe it would be nice to add the same image for the other examples?